end-to-end-tests/sdk-set-user-id.test.js

```diff
@@ -0,0 +1,146 @@
+import { ErrolTestClient } from './errolclient';
+import { launchServer, createChromeWebDriver } from './test-utils';
+
+let killServer = null;
+let chromeDriver = null;
+
+beforeAll(() => {
+  return launchServer()
+    .then(killFunc => {
+      killServer = killFunc;
+    })
+    .then(() => createChromeWebDriver())
+    .then(driver => {
+      chromeDriver = driver;
+    })
+    .then(async () => {
+      await chromeDriver.get('http://localhost:3000');
+      await chromeDriver.wait(() => {
+        return chromeDriver
+          .getTitle()
+          .then(title => title.includes('Test Page'));
+      }, 2000);
+
+      return chromeDriver.executeAsyncScript(() => {
+        const asyncScriptReturnCallback = arguments[arguments.length - 1];
+
+        let deleteDbRequest = window.indexedDB.deleteDatabase(
+          'beams-1b880590-6301-4bb5-b34f-45db1c5f5644'
+        );
+        deleteDbRequest.onsuccess = asyncScriptReturnCallback;
+        deleteDbRequest.onerror = asyncScriptReturnCallback;
+      });
+    })
+    .then(async () => {
+      const errolClient = new ErrolTestClient(
+        '1b880590-6301-4bb5-b34f-45db1c5f5644'
+      );
+      const response = await errolClient.deleteUser('cucas');
+      expect(response.statusCode).toBe(200);
+    });
+});
+
+test('SDK should set user id with errol', async () => {
+  await chromeDriver.get('http://localhost:3000');
+  await chromeDriver.wait(() => {
+    return chromeDriver.getTitle().then(title => title.includes('Test Page'));
+  }, 2000);
+
+  const initialDeviceId = await chromeDriver.executeAsyncScript(() => {
```

Comment:
The device ID never changes - so maybe just 'deviceId'?
```suggestion
  const deviceId = await chromeDriver.executeAsyncScript(() => {
```

Reply:
Should I change the variable name from `initialDeviceId` to `deviceId` also in the `sdk-start-test.js`?

Reply:
The first test I guess - the second one makes sense

Reply:
Fixed.